src/Faithlife.Utility/CollectionUtility.cs

```diff
@@ -43,7 +43,7 @@ public static void AddRange<T>(this ICollection<T> collection, IEnumerable<T> it
 		/// <param name="value">The value to add. May be null.</param>
 		public static void AddIfNotNull<T>(this ICollection<T> collection, T value)
 		{
-			if (value != null)
+			if (value is object)
```

Comment:
For the reasons given in this other Twitter post: https://twitter.com/jaredpar/status/1115019017297596416

I admit it takes a little getting used to, but it composes better in other expressions and I think there's a benefit in avoiding negated parenthesised expressions.

Reply:
I don't find that checking whether "something is an object" is unintuitive. It seems like a logical opposite of "is null" to me.

Reply:
I liked the suggestion of `x aint null` in the comments lol. 
It will take a while to get used to for sure. 
The more I thought about it last night the more it grew on me though. Mostly because you are normally doing the `x is object` check followed by an action on `x` which makes logical sense.